anime_downloader/sites/animesvostfr.py

```diff
@@ -0,0 +1,262 @@
+from anime_downloader.sites.anime import Anime, AnimeEpisode, SearchResult
+from anime_downloader.sites import helpers
+import logging
+import re
+
+from requests.exceptions import HTTPError
+from anime_downloader.config import Config
+from anime_downloader.extractors import get_extractor
+
+logger = logging.getLogger(__name__)
+
+class AnimesVOSTFR(Anime, sitename='animesvostfr'):
+       
+        sitename = 'animesvostfr'
+        QUALITIES = ['360p', '480p', '720p', '1080p']
+        DOMAIN = "https://www1.animesvostfr.net/"
+
+        @classmethod
+        def search(cls, query):
+            
+            soup = helpers.soupify(helpers.get(cls.DOMAIN, params={'s' : query}))
+            
+            results = []
+            
+            for v in soup.select('div.movies-list > div.ml-item'):
+                t = v.h2.text
+                u = v.a['href']
+                p = v.img['src']
+                
+                lang = v.select_one('div.language').text.upper()
+                
+                #logger.debug( "results: {}".format(lang) )
+                
+                eps = v.select_one('span.mli-quality').text.upper()
+                
+                #logger.debug( "results: {}".format(eps) )
+                
+                lang_dict = {
+                'VOSTFR':f' (VOSTFR {eps})',
+                'VF':f' (VF {eps})',
+                'VO':f' (VO {eps})'
+                }
+                selected_lang = f' (V? {eps})' #default lang
+                for a in lang_dict:
```

Comment:
This loop is confusing. Add a comment to explain what it is doing.

Reply:
OK i do it, it's for add part at the title like:
(VOSTR 12/12)
(VF 12/12)
(VF MOVIE)